### PR TITLE
Update Password rule pattern

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- applied pattern PW = xxx for password rule

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest 
- [x] Test Benchmark

### CODE REVIEW

File: credsweeper/rules/config.yaml [hunk 1]
```diff
@@ -246,13 +246,13 @@
   severity: medium
   type: keyword
   values:
-    - pass|pwd
+    - pass|pw(d|\b)
```
Comment (csh519) [inline credsweeper/rules/config.yaml:249]: Using `\b` for checking word boundary is good :eyes: 
I hope this change doesn't make much FPs in real world..

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -62,7 +62,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
+        ref: pw
```
Comment (csh519) [inline .github/workflows/benchmark.yml:66]: Please change this field when https://github.com/Samsung/CredData/pull/36 is merged.

Comment (babenek) [inline .github/workflows/benchmark.yml:66]:
```suggestion
        repository: Samsung/CredData
```

**Review by csh519 — APPROVED** [review]
Thank you for updating `Password` rule.

LGTM :+1:

**Review by iuriimet — APPROVED** [review]

**Pull request merged**